package.json

```diff
@@ -306,10 +306,18 @@
           "command": "one.explorer.showExtra",
           "when": "false"
         },
+        {
+          "command": "one.toolchain.install",
+          "when": "false"
+        },
```

Comment:
Um, I want to show this command in command palette. Because it is very useful to install new toolchain. :)

Reply:
I see. Let me change `faler` to `true`.